### PR TITLE
allow warning for namelist validation

### PR DESCRIPTION
**Pull request by jmccreight**

### CODE REVIEW

File: wrfhydropy/core/ensemble.py [hunk 1]
```diff
@@ -286,8 +286,9 @@ def visit(path, key, value):
     def compose(
         self,
         symlink_domain: bool=True,
-        force: bool = False,
-        rm_members_from_memory: bool = True
+        force: bool=False,
+        check_nlst_warn: bool=False,
```
Comment (jmills-ncar) [inline wrfhydropy/core/ensemble.py]: update docstring with new argument

File: wrfhydropy/core/ioutils.py [hunk 2]
```diff
@@ -177,16 +186,24 @@ def remap_nlst(nlst):
...
 
     if ignore_restarts:
         hydro_exempt_list = hydro_exempt_list + ['restart_file']
         hydro_exempt_list = hydro_exempt_list + ['nudginglastobsfile']
         hrldas_exempt_list = hrldas_exempt_list + ['restart_filename_requested']
 
-    def check_nlst(nlst, file_dict):
-
+    def check_nlst(
+        nlst,
```
Comment (jmills-ncar) [inline wrfhydropy/core/ioutils.py]: update docstring with arguments

**Comment by jmills-ncar** [comment]
Just a couple of minor docstring changes

File: wrfhydropy/core/ensemble.py (hunk 1 above)
Comment (jmccreight) [reply wrfhydropy/core/ensemble.py]: There it is...

File: wrfhydropy/core/ioutils.py [hunk 3]
```diff
@@ -177,15 +186,27 @@ def remap_nlst(nlst):
...
-    def check_nlst(nlst, file_dict):
+    def check_nlst(
+        nlst,
+        file_dict,
+        warn: bool=False
+    ):
+        """
+        Check the paths in the namelist.
+        Args:
+            nlst: The namelist to check.
```
Comment (jmccreight) [inline wrfhydropy/core/ioutils.py]: I have documented the previously un-documented.

**Pull request merged**